lib/msal-browser/src/naa/TokenRequest.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export type TokenRequest = {
```

Comment:
Can you extend one of the existing request types? e.g.

```
BaseAuthRequest & {
newProperty: string 
}
```

Reply:
This is not an interface owned by us... it's defined by the bridge.